.gitignore

```diff
@@ -56,4 +56,4 @@ pip-wheel-metadata
 .DS_Store
 
 # setuptools-scm generated version file
-asdf/version.py
+asdf/_version.py
```

Comment:
This is _technically_ a breaking change but I can't find any code that refers to this module outside of the asdf library, so I don't expect any problems.

Reply:
Its essentially the same file, it just brings it in line with `setuptools_scm` convention. All it provides is `__version__`. Which we might want to update everything in `asdf` which was importing this to use instead?

Reply:
I just mean that previously it was possible for users to do this:

```
from asdf.version import version
```

and now it's not, but I don't think that's something we need to worry about since the API we intended to expose was `asdf.__version__`